src/backend/routeHandler.ts

```diff
@@ -0,0 +1,42 @@
+import {PluginConfig, PluginRouteOptions} from '../@types/plugin';
+import bodyParser from 'body-parser';
+
+export = function configureRoutes(options: PluginRouteOptions<PluginConfig>): void {
+
+  options.router.use(bodyParser.json());
+  options.router.get('/getOgmaConfiguration', async (req, res) => {
+    try {
+      const LKEConfiguration = await options.getRestClient(req).config.getConfiguration();
+      res.contentType('application/json');
+      if (LKEConfiguration.isSuccess()) {
+        res.status(200);
+        res.send(JSON.stringify(LKEConfiguration.body.ogma));
+      } else {
+        res.status(412);
+        res.send(JSON.stringify({status: 412, body: {message: LKEConfiguration.body.message}}));
+      }
+    } catch (e) {
+      res.status(412);
+      res.send(JSON.stringify({status: 412, body: e}));
+    }
+  });
+  options.router.get('/getVisualizationConfiguration/:id&:sourceKey', async (req, res) => {
```

Comment:
this is a very non standard way to get parameters, instead, let's pass them as querystring.